.gitignore

```diff
@@ -1,2 +1,3 @@
 Thumbs.db
 .github/
+test.md
```

Comment:
Hi George,

Thank you very much for your support and sorry for the mess especially in GitHub and GitHub Flow, I will focus on having cleaner and more understandable pull requests in future projects.

At the moment I have a doubt, apparently, there is a file that does not appear in the Node_modules according to the linters, but reviewing the folder I found the file; looking for some solution I found a question in discourse that was solved by changing the linters.yml file. I would like to know if I would have to change it or if for the moment I keep it like that.

Thank you for all the feedback given so far.

Reply:
@Pyonguk , Sorry for the late response, it's a result of time zone differences.

Please, whatever you do, do not change the configuration of linters.yml file. You're already doing well and making progress. If you feel you're stuck, you can get help to unblock you through any of the means provided by Microverse. I am also available in case you need me.

Reply:
Don't worry, anyway thank you for your feedback.

I think I already finish with everything, all my linters work fine at local and I try to re-install the Stylint but the problem remains (A file is still missing.). 

Apart from this problem, I would like to know if in the README I have to put something in the "Getting Started" section since I think that I should not have to download anything to access the repository without problems. Although if they are more specific downloads, like installing Node.js or how is the process to download and install the linters let me know.

Thanks again.

Reply:
"Getting Started" section is about how a user can set up a local copy of your project. I think it's applicable to your project and you should include.

Reply:
Done, I created a branch, made the changes I considered necessary to the README, and merge it.

Reply:
What should I do next?

Reply:
When you've made the required changes, you need to re-submit the project